src/store/products/products.types.ts

```diff
@@ -0,0 +1,45 @@
+// Entities
```

Comment:
Since this file contains no imports, yet exports to hooks and routes, we can tell that this is primary to the approach. Indeed this defines the types we will be using for our various tasks, the key feature of typescript, the purpose of which is to reduce errors by requiring code to match type with type?